File: program/program/check-alphabet/CheckAlphabet.c

//CheckAlphabet.c

#include <stdio.h>

int main()
{
    char ch;
    scanf("%c",&ch);
    if((ch>='A' && ch<='Z') || (ch>='a' && ch<='z')) printf("Alphabet\n");
    else printf("Not Alphabet\n");
    return 0;
}

File: program/program/check-alphabet/CheckAlphabet.kt
fun main() {
    print("Input : ")
    println()
    var input = readLine()

    println("Is $input is an alphabet : " + CheckAlphabet(input!!))

}

private fun CheckAlphabet(inputStr: String): Boolean {
    val c = inputStr.first()
    if (c in 'a'..'z' || c in 'A'..'Z')
        return true
    else
        return false
}
